### DISCUSSION TITLE
你好，我想问一下，为什么法学引注手册生成的参考文献内容是不符合格式要求的

### DISCUSSION THREAD

**Discussion by ganwufeizhai** [zotero-chinese/wiki#229] [求助 / Q&A] [answered]
你好，我想问一下，为什么法学引注手册生成的参考文献内容是不符合格式要求的
![Uploading 3163106FAC09AFA274364B5BF4283901.png…]()

**Comment by northword** [comment]
无法查看图片，或请提供格式要求和实际结果，并简要描述不符合要求的地方。

上传图片需要等上传完成再提交评论。

**Reply by ganwufeizhai** [reply]
大概如下，请您看看

常用的文献参考格式

肖鹏：《我国刑事法网的基本现状及发展趋势——兼评积极主义刑法观》，《河北法学》，2021年第10期，第149–165页。
肖鹏：《我国刑事法网的基本现状及发展趋势——兼评积极主义刑法观》，《河北法学》，2021年第10期，第149–165页。
法学引注手册（多语言）

肖鹏, 我国刑事法网的基本现状及发展趋势——兼评积极主义刑法观, 39 河北法学 149 (2021).
〔1〕
肖鹏, 我国刑事法网的基本现状及发展趋势——兼评积极主义刑法观, 39 河北法学 149 (2021).

**Reply by northword** [reply, marked as answer]
1. 确认用的是从 https://zotero-chinese.com/styles/manual-of-legal-citation-multi-lingual/ 下载的最新的 csl
2. 确认条目语言是 zh 或 zh-CN 。

**Reply by ganwufeizhai** [reply]
已解决，非常感谢
